lib/boxes.js

```diff
@@ -15,6 +15,14 @@ async function deleteBoxByTitle(title) {
   return deleteBox;
 }
 
+async function deleteItemByName(title, item) {
+  const deletedItem = await collection("boxes").updateOne(
+    { title },
+    { $pull: { items: { $in: [item] } } }
```

Comment:
What happens if you write it like this. 
```suggestion
    { $pull: { items: item } }
```

Reply:
Same, same. Without `$in` and without the square brackets, it delete the whole object.

Reply:
Nico found the problem - I have used the wrong function at server.js 🤦🏼‍♀️